### ISSUE TITLE
; (semi-colon) in TXT records causes the remainder of the record to be dropped on import

### ISSUE THREAD

**Issue by avsdev-cw** [pacoorozco/probind#67]
**Describe the bug**
`;` (semi-colon) in TXT records cause the remainder of the record to be dropped, eg:
```
_dmarc              IN TXT    "v=DMARC1; p=quarantine; rua=mailto:postmaster@domain.local; ruf=mailto:postmaster@comain.local; fo=1; adkim=r; aspf=r; pct=100; sp=quarantine"
```
Gets imported as:
```
_dmarc              IN TXT    "v=DMARC1;
```

The issue lies with the parser, it hits `;` and assumes it is a comment marker, even though it is within a quote

**To Reproduce**
Import an existing zone file with TXT records with `;` (semi-colon's) in them.

**Comment by pacoorozco** [comment]
Hi @avsdev-cw 

I've worked in a patch that is available for testing ay `change-file-dns-parser` branch. **Can you test it?**

**Comment by pacoorozco** [comment]
Fixed on `v0.10.0`.
